JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
-DevExtreme Data Editors provide a&nbsp;powerful and seamless way to&nbsp;perform data validation. In&nbsp;this demo, editors are collected in&nbsp;an&nbsp;HTML&nbsp;form. Each editor is&nbsp;accompanied by&nbsp;the Validator component validating the editor against a&nbsp;set of&nbsp;specified rules. Editors are validated each time their value is&nbsp;changed. All validation errors are displayed in&nbsp;the ValidationSummary component.
+This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on the page with enabled data validation. In this demo, the editors are collected in an HTML form.
```

Comment:
> In this demo, the editors are **collected** in an HTML form.

Could you clarify what do you mean by "collected" here? Grouped?

> a group of editors on **the** page with enabled data validation

On **any** page with enabled data validation. The only necessary condition is "enabled data validation", and it could any page where it is possible to enable data validation.

```suggestion
This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on a page with enabled data validation. In this demo, the editors are collected in an HTML form.
```